### PR TITLE
Update TrustGraphs.md

### PR DESCRIPTION
**Pull request by susanne-guth-orlowski**
Added a first small section about trusted issuers list.

### CODE REVIEW

File: website/docs/design-patterns/TrustGraphs.md [hunk 1]
```diff
@@ -9,10 +9,19 @@ import Disclaimer from '../\_disclaimer.mdx';
...
+
+To combat this, trusted GHG emissions claim issuers must be accredited and represented digitally. This is where the trusted issuers list becomes essential. It serves as a dynamic electronic register that tracks all issuers who have received accreditation for CO2 emissions calculations from a recognized authority, such as the Global Battery Alliance. This ensures that only verified and trustworthy entities can issue these critical credentials, maintaining integrity and trust in the system. Such a trusted issuers list can for example be implemented by adding all DIDs of trusted GHG emissions claim issuers to a public blockchain, such as Ethereum. 
+
+When a GHG emissions VC holder presents their credential to a verifier, both parties use a digital wallet for this exchange. This software enables its owner to securely acquire, store, manage, and verify VCs. The verifier’s wallet is responsible for automatically scrutinizing the incoming VC for authenticity. There are several checks a VC must pass, but here we will focus on issuer verification.
```
Comment (Fak3) [inline website/docs/design-patterns/TrustGraphs.md:21]: From my experience, term "digital wallet" is very ambiguous, different people imply very different things by this. Furthermore, presentation and verification of a credential can perfectly be performed without a wallet, but by http api requests sequence. So I would suggest to reword this section saying that verifier software verifies, validates and checks authenticity.

**Review by Fak3 — CHANGES_REQUESTED** [review]

**Review by onthebreeze — COMMENTED** [review]
The UNTP has been designed to avoid the need for wallets.  Can we discuss this one further please?

**Review by onthebreeze — COMMENTED** [review]
there is nothing ot prevent use of wallets (eg verifiers can fetch and store in a wallet) or holders can present from wallets.  But we prefer not to depend on wallets.

**Comment by philarcher** [comment]
The text proposed is perfectly fine IMO, however, the term "recognized authority" perhaps needs considering. Recognized by whom? If we're talking individuals then, well, you trust who you trust. That's not an engineering task. But for UNTP, do we need to contemplate some sort of register of trusted parties? Such a register would, I imagine, cite the likes of ILAC and IAF as roots of trust, but that should not exclude others like the GBA or Rainforest Alliance.

Even then, it's not really enough. Sticking with CO2 emissions, if I have two different assessments of a product's CO2 footprint, one linked from the DPP and one I found elsewhere, and they differ, well, who's to say which one's right? Manufacturers will go to the GHG emission calculator that gives them the best result, whether it's true or not. An independent one might give an assessment that, as an individual, I might trust more. In the real world, it's likely that the independent assessment will be less favourable. 

Anything like a register of recognized authorities would be a big overhead and one we almost certainly want to avoid if at all possible but I fear it may be unavoidable.

**Comment by onthebreeze** [comment]
modifed suzanne's text and created a new PR.  Therefore closing this one.

**Pull request closed without merging**